cli/tests/pcluster/config/defaults.py

```diff
@@ -282,6 +325,72 @@ class DefaultDict(Enum):
 }
 
 
+DEFAULT_CLUSTER_HIT_CFN_PARAMS = {
```

Comment:
should we have a COMMON? see previous comment

Reply:
What would be the advantages brought by this optimization? I'd keep the freedom of having completely separated dicts, since also common parameters could have a different default value, like `disable_hyperthreading`.